Tests/bubbleSort/bubbleSort.c

```diff
@@ -28,7 +28,9 @@ int main(int argc, char *argv[])
     int *in = get_input(SIZE);
 
     // bubble sort
+#ifdef TRACING
```

Comment:
I put those directives there when I was debugging tikSwap because they appeared to be segfaulting when they were included. Having those function symbols is not important to the unit tests so I didn't include the directives anywhere in the unit test flow. If they're needed somebody can add them at it through their own framework

Reply:
This isn't here as the master copy of the unit test. If the code is here we should enable it. If we don't want it it should be gone. If you want to add an option to cmake thats fine, but if the code is useless I want it removed.

Reply:
done